source/game_sa/Camera.cpp

```diff
@@ -964,7 +964,10 @@ void CCamera::VectorTrackLinear(CVector* trackLinearStartPoint, CVector* trackLi
 
 // 0x50D240
 void CCamera::AddShakeSimple(float duration, int32 type, float intensity) {
-    plugin::CallMethod<0x50D240, CCamera*, float, int32, float>(this, duration, type, intensity);
+    m_fShakeIntensity = intensity;
+    m_nShakeType = type;
+    m_fStartShakeTime = CTimer::GetTimeInMS();
```

Comment:
We want a compiler, not a complainer! So use explicit casts.
```suggestion
    m_fStartShakeTime = (float)CTimer::GetTimeInMS();
```